stage0_bin/supply-chain/imports.lock

```diff
@@ -28,6 +28,12 @@ so they'll at least be forthcoming if they try to implement something
 cryptographic. When in doubt, please ask an expert."""
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
+[[audits.google.audits.bit_field]]
```

Comment:
What happened here?

Reply:
`bit_field` and `bitflags` are dependencies of `x86_64`. I think that `bitflags` was the last unvetted dependency of `x86_64`, so `cargo vet certify` automatically imported `bit_field` that has already been vetted into the lock file. I am not 100% sure that this is the logic, but seems to be a pattern when certification of one crate causes others to be imported into the lock file.